src/Spinner.class.ts

```diff
@@ -57,7 +59,7 @@ export class Spinner extends EventEmitter {
   #spinner: cliSpinners.Spinner;
   #text = "";
   #prefix = "";
-  #color: (stdout: string) => string;
+  #color: string;
```

Comment:
```suggestion
  #color: Color;
```
May we can do this so we can remove the cast below ? (`as Color`)